main_test.go

```diff
@@ -15,22 +15,37 @@ import (
 // unit tests of individual functions
 var _ = Describe("CfDownload", func() {
 	var args []string
+	var paths []string
 
-	BeforeEach(func() {
-		args = make([]string, 7)
-	})
-
-	Describe("Test Flag functionality", func() {
+	Describe("Test ParseArgs functionality", func() {
 
 		Context("Check if overWrite flag works", func() {
 			It("Should set the overwrite_flag", func() {
+				args = make([]string, 4)
```

Comment:
I totally understand why you allocated the arrays this way, but I think you have a better option. I found [this](https://blog.golang.org/go-slices-usage-and-internals) in the official docs:

>Or, you can have the compiler count the array elements for you: `b := [...]string{"Penn", "Teller"}`